python/lsst/fgcmcal/fgcmBuildFromIsolatedStars.py

```diff
@@ -176,7 +176,7 @@ def setDefaults(self):
 
         source_selector.flags.bad = []
 
-        source_selector.signalToNoise.minimum = 11.0
+        source_selector.signalToNoise.minimum = 9.0
```

Comment:
Why this seemingly arbitrary change in minimum S/N?

Reply:
Ah. Good question!  The issue is that the compensated tophats are actually computing the noise properly, so they have slightly lower s/n than the apertures and this (empirically) gives the same number of stars, so is an equivalent selection.